app/code/Magento/Customer/Model/AttributeChecker.php

```diff
@@ -8,7 +8,6 @@
 use Magento\Customer\Api\AddressMetadataInterface;
 use Magento\Customer\Api\AddressMetadataManagementInterface;
 use Magento\Customer\Model\Metadata\AttributeResolver;
-use Magento\Framework\App\Helper\Context;
```

Comment:
Hi @chris-pook, please check https://github.com/magento/magento2/commit/a872f4061695b91e923d1c1abb87a0a50e18c77d#diff-fe978939bb978f6074e10338e0b14791

I believe the fix is not correct, helper context is supposed to be injected, not customer model context.

At least parent class expects `Magento\Framework\App\Helper\Context`.

Could you please replace import with `Magento\Framework\App\Helper\Context as HelperContext` instead and change other places in class accordingly?

Reply:
Sorry @orlangur , you've lost me.. `Customer\Model\AttributeChecker` (where my change is) has no parent class. This change I have made is in `Customer\Model\AttributeChecker` where there is a `use` statement for `Magento\Framework\App\Helper\Context` which is never used in the class. I could alias this to `HelperContext` but it is not used so better to remove I think? Unless I'm missing something here?

Reply:
@chris-pook, my bad, I was opening `AttributeChecker.php` in my IDE and didn't notice I'm checking helper and not model with the same.

Looks good to me now after your explanation :)

Reply:
Ah good stuff! Thanks